Documentation/docs/contributing/enforcement.md

```diff
@@ -70,8 +70,9 @@ In this document, the term "the originator" refers to the person(s) that have
 violated this Code of Conduct and originated an incident; the term "the
 affected party(ies)" refers to the person(s) that have suffered the attack of "the
 originator" under the forms described in the
-[Unacceptable Behavior](CODE_OF_CONDUCT.md#unacceptable-behavior) section.
+[Unacceptable Behavior](code_of_conduct.md) section.
```

Comment:
Did you remove the anchor on purposed ?

Reply:
Yes -- cross-document references were not supported by Myst.